cicd/resources/model.py

```diff
@@ -271,3 +271,22 @@ def finish_initial_load(self, source_id):
         if response.status_code != 200:
             logger.info(f'Response is {response.content}')
             raise RuntimeError('Response is not 200. Exiting')
+
+    def is_model_deployed(self):
```

Comment:
I believe it is a good idea to pass the `repo name` or `repo id` as an argument here. Why are we trying to loop through all the repos?

Reply:
I have passed the repo_id